src/View/Helper/HtmlHelper.php

```diff
@@ -137,7 +137,7 @@ public function meta(array|string $type, array|string|null $content = null, arra
     {
         if (is_string($type)) {
             $types = [
-                'csrfToken' => ['name' => 'csrf-token'],
+                'csrf-token' => ['name' => 'csrf-token'],
```

Comment:
I'm assuming there were applications using `csrfToken` and providing the `content` parameter as well? Couldn't we end up breaking a different set of applications with a rename?

Reply:
> I'm assuming there were applications using csrfToken and providing the content parameter as well?

Yes, the problem was they expected a meta tag with name `csrfToken` but ended up with `csrf-token` instead.

> Couldn't we end up breaking a different set of applications with a rename?

That would be possible only if they were using some other content value than the one we have in the request attribute, but I don't see what good that would do.

This issue right now isn't the value but the name inflection.

Reply:
> That would be possible only if they were using some other content value than the one we have in the request attribute, but I don't see what good that would do.

Me neither, but we've gotten issues for all sorts of changes over the years :shrug:

Reply:
FYI, this is the code:

`$this->Html->meta('csrfToken', $this->request->getAttribute('csrfToken'))`